doc/nrf/security/crypto/crypto_supported_features.rst

```diff
@@ -127,6 +190,111 @@ The options are grouped by Series and drivers available for the device Series, a
                  - Supported
                  - Supported
                  - Supported
+               * - XChaCha20
+                 - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_XCHACHA20`
+                 - Supported
+                 - Supported
+                 - Supported
+               * - HSS Public Key
+                 - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_HSS_PUBLIC_KEY`
+                 - Supported
+                 - Supported
+                 - Supported
+               * - LMS Public Key
+                 - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_LMS_PUBLIC_KEY`
+                 - Supported
+                 - Supported
+                 - Supported
+               * - XMSS Public Key
+                 - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_XMSS_PUBLIC_KEY`
+                 - Supported
+                 - Supported
+                 - Supported
+               * - XMSS-MT Public Key
+                 - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_XMSS_MT_PUBLIC_KEY`
+                 - Supported
+                 - Supported
+                 - Supported
+               * - ML-DSA-44
+                 - :kconfig:option:`CONFIG_PSA_WANT_ML_DSA_KEY_SIZE_44`
+                 - Supported
+                 - Supported
+                 - Supported
+               * - ML-DSA-65
+                 - :kconfig:option:`CONFIG_PSA_WANT_ML_DSA_KEY_SIZE_65`
+                 - Supported
+                 - Supported
+                 - Supported
+               * - ML-DSA-87
+                 - :kconfig:option:`CONFIG_PSA_WANT_ML_DSA_KEY_SIZE_87`
+                 - Supported
+                 - Supported
+                 - Supported
+               * - ML-DSA Key Pair Import
+                 - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_ML_DSA_KEY_PAIR_IMPORT`
```

Comment:
We have the _IMPORT and other options here but for the ECC keys lets say these are placed in another section which is called "ECC operations below". I think that it is better to place these in their own section instead of placing them here. 

Also even though this is not relevant to this PR there is not "CONFIG_PSA_WANT_KEY_TYPE_ECC_KEY_PAIR" or "CONFIG_PSA_WANT_KEY_TYPE_RSA_KEY_PAIR". These have been removed and replaced and IMPORT/GENERATE/etc scheme that we have for the ML_DSA keys.

Reply:
Created NCSDK-34817 for this <ECC/RSA>_KEY_PAIR unresolved Kconfig issues